sbin/build.sh

```diff
@@ -837,6 +843,9 @@ getFirstTagFromOpenJDKGitRepo() {
   else
     echo "$firstMatchingNameFromRepo"
   fi
+
+  # Restore pwd
+  cd "$savePwd"
```

Comment:
You could probably just do `cd $OLDPWD` (or even `cd -`) instead of using a new variable to store the location.

Reply:
Although that would work, I am cautious of using a logic that changes to the "previous", as what if someone later adds some intermediate function call that changes to some directory etc... this logic in this function would then not achieve the desired "spec", which is change back to the directory in which the getFirstTagFromOpenJDKGitRepo() function entered, cd - may not achieve that.

Reply:
We can't use cd - as that echo's the pwd, which would then end up in the jdk version!

Reply:
Not sure how reliant $OLDPWD is if the previous directory before was the same directory

Reply:
Given the questions and risk of someone just adding an extra cd in this function, i'd rather leave it as it is